kong/runloop/handler.lua

```diff
@@ -110,10 +110,15 @@ do
 
     sort(routes, function(r1, r2)
       r1, r2 = r1.route, r2.route
-      if r1.regex_priority == r2.regex_priority then
+
+      local reg_prio1 = r1.regex_priority or 0
```

Comment:
Can `regex_priority` ever be `ngx.null` rather than `nil`?

Reply:
Not in the proxy, where we purposefully don't use the `opts.nulls` option.